crates/types/src/services/txpool.rs

```diff
@@ -180,7 +180,7 @@ pub enum TransactionStatus {
     /// Transaction was successfully included in a block
     Success {
         /// Included in this block
-        block_id: BlockId,
+        block_height: BlockHeight,
```

Comment:
If there's a possibility of reorgs in the future, would only using block height on the api be problematic?

Reply:
With a shared sequencer, I don't see how we can occur fork with several valid blocks for the same height. It is possible to have invalid blocks and valid blocks, but we will skip invalid due to invalidity and commit only a valid one.

Reply:
so if there are several proposed block ids for the same block height, and a user tries to see the block containing their tx, the tx might not exist in the block that actually gets accepted right?

Reply:
Yes, the transaction has `Committed` status only when the corresponding block is finalized by the shared sequencer.